x-pack/plugins/security_solution/public/management/pages/policy/test_utils/index.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
```

Comment:
just for the sake of clarity I'd prefix all these functions with `getMick` and not only `get`  as it can be confused by editors that autocomplete based on analyzing code.